trex-operator/config/manager/manager.yaml

```diff
@@ -43,6 +43,14 @@ spec:
               port: 6789
             initialDelaySeconds: 5
             periodSeconds: 10
+          # operator-sdk is not implementing startup probe up to now, using liveness probe endpoint
+          # while this feature is developed
+          startupProbe:
+            httpGet:
+              path: /healthz
```

Comment:
Do you know if the /healthz endpoint is accessible before the /readyz ?

Reply:
At least in the source code of operator-sdk, the `/healthz` endpoint is defined just before `/readyz`: https://github.com/operator-framework/operator-sdk/blob/master/internal/cmd/helm-operator/run/cmd.go#L177-L184, but this is prior to launching the logic, so both should be accessible anyway without any issue.